src/index.html

```diff
@@ -1,13 +1,46 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        src="./images/imac.jpeg" 
+        alt="screen" 
+        class="card__img"
+      >
+      <div class="card__name-item">
+        <p>APPLE A1419 iMac 27&#34; Retina 5K Monoblock &#40;MNED2UA&#47;A&#41;</p>
+      </div>
+      <div class="card__name-article">Product code&#58; 195434</div>
+      <div class="card__card-rate">
+        <div class="card__stars">
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+        </div>
+        <div class="card__rate-reviews">
```

Comment:
```suggestion
        </div>
        
        <div class="card__rate-reviews">
```

Reply:
Thank you for your review. I would like to ask, I need to divide div because, it is separate elements?